sharding/src/main/resources/application.conf

```diff
@@ -96,5 +100,11 @@ shard-bench {
   pings-per-second = 800
   number-of-shards = 100
   total-nodes = 1
+  
+  
+  # actor counting scalability -------------
+  add-actors-batch = 100
+  add-actors-interval = 5.seconds
+  # end of actor counting scalability ------
```

Comment:
that will take forever, a single node should be able to host several 100,000 entities, but I guess it's just a matter of adjusting those